src/config/types.ts

```diff
@@ -212,6 +212,9 @@ export enum AggregationOperations {
   Median = 'median',
 }
 
+export const isStatistic = (stat: string): stat is AggregationOperations =>
```

Comment:
We could have a function for this, but I personally feel `stat in AggregationOptions` is readable enough, and I'm pretty sure TS is smart enough to apply its own type predicate if it sees this. This is up to discussion however, and you are free to keep this if you think its better :+1: